specifyweb/frontend/js_src/lib/components/Header/Notifications.tsx

```diff
@@ -134,6 +134,21 @@ export function Notifications({
           count: notifications.length,
         })
       : notificationsText.notificationsLoading();
+
+  const onClearAll = () => {
+    if (notifications !== undefined) {
+      notifications.forEach((notification) => {
+        ping('/notifications/delete/', {
+          method: 'POST',
+          // eslint-disable-next-line @typescript-eslint/naming-convention
+          body: formData({ message_id: notification.messageId }),
+          errorMode: 'dismissible',
+        }).then(f.void);
```

Comment:
Why is this `.then(f.void)` needed? The Promise containing the `ping` should already be considered a void function, because it does not return anything (undefined).